staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -1142,3 +1142,149 @@ type Fields struct {
 	// The exact format is defined in k8s.io/apiserver/pkg/endpoints/handlers/fieldmanager/internal
 	Map map[string]Fields `json:",inline" protobuf:"bytes,1,rep,name=map"`
 }
+
+// TODO: Table does not generate to protobuf because of the interface{} - fix protobuf
+//   generation to support a meta type that can accept any valid JSON.
```

Comment:
Yes I think it will be.  Since we reject proto now, adding it is like widening validation without the possibility that a client gets the widening without asking.  Also we don't store these types on disk.